build/docs/build_docs.py

```diff
@@ -120,8 +120,9 @@ def format_as_document(magic, uid_base : str) -> MagicReferenceDocument:
     uid = f"{uid_base}.{safe_name}"
     metadata = {
         'title': f"{magic_name} (magic command)",
+        'author': rmshaffer,
```

Comment:
```suggestion
        'author': 'rmshaffer',
```
You'll likely also want an `ms.author` here.